src/frontend/src/session.rs

```diff
@@ -145,8 +145,12 @@ impl FrontendEnv {
         let worker_node_manager = Arc::new(WorkerNodeManager::mock(vec![]));
         let meta_client = Arc::new(MockFrontendMetaClient {});
         let hummock_snapshot_manager = Arc::new(HummockSnapshotManager::new(meta_client.clone()));
-        let query_manager =
-            QueryManager::new(worker_node_manager.clone(), hummock_snapshot_manager);
+        let compute_client_pool = Arc::new(ComputeClientPool::new(1024));
```

Comment:
```
constant COMPUTE_CLIENT_POOL_SIZE = 1024
```

Reply:
Maybe we can simply use u64::MAX here? This is indeed not a cache, we want to reuse as many connections as possible 🤣